buildHelpers/writeCjsEntryFile.js

```diff
@@ -0,0 +1,16 @@
+const fs = require('fs-extra');
```

Comment:
Instead of generating and writing this file with fs, could this be a static file that is just copied over? `name` looks like it's always index?